arangod/Aql/Ast.h

```diff
@@ -416,6 +416,7 @@ class Ast {
   void validateAndOptimize();
 
   /// @brief determines the variables referenced in an expression
+  static void getReferencedVariables(AstNode const*, std::unordered_set<Variable const*>&);
```

Comment:
I don't see a value in this given that the other method already exists.

Reply:
without this there is a build failure http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-linux/3959/

Reply:
you have to merge the recent `devel` into your branch then

Reply:
that branch was made from recent devel, and this is a build failure in community, not enterprise

Reply:
current devel has only 1 version of getReferencedVariables function and compiles without any errors: https://github.com/arangodb/arangodb/blob/devel/arangod/Aql/Ast.h
Probably you have to pull the recent devel into your branch once again.

Reply:
the branch was based on a clean devel clone as of this morning, as per the git history